docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md

```diff
@@ -0,0 +1,211 @@
+---
+id: opensearch-exporter
+title: "Opensearch Exporter"
+sidebar_label: "Opensearch"
+---
+
+The Zeebe Opensearch Exporter acts as a bridge between
+[Zeebe](https://camunda.com/platform/zeebe/) and [Opensearch](https://opensearch.org), by
+exporting records written to Zeebe streams as documents into several indices.
+
+## Concept
+
+The exporter operates on the idea that it should perform as little as possible on the Zeebe side of
+things. In other words, you can think of the indexes into which the records are exported as a
+staging data warehouse: any enrichment or transformation on the exported data should be performed by
+your own ETL jobs.
+
+To simplify things, when configured to do so, the exporter will automatically create an index per
+record value type (see the value type in the Zeebe protocol). Each of these indexes has a
+corresponding pre-defined mapping to facilitate data ingestion for your own ETL jobs. You can find
+those as templates in this module's resources folder.
+
+> **Note:** the indexes are created as required, and will not be created twice if they already
+> exist. However,
+> once disabled, they will not be deleted, that is up to the administrator. Similarly, data is never
+> deleted by
+> the exporter, and must, again, be deleted by the administrator when it is safe to do so.
```

Comment:
🔧 This note format `> **Note**` is a [specific Github notation](https://github.com/orgs/community/discussions/16925). For Docusaurus, I'd recommend switching to the [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions):

```
:::note

here's my note

:::
```